### PR TITLE
Updated German translation

### PR DESCRIPTION
**Pull request by dev2adventure**
General improvement of the German translation, especially in terms of the translation for "Parent" and "Child".

### CODE REVIEW

File: src/locales/de.ts [hunk 1]
```diff
@@ -66,19 +66,19 @@ export const translations: Translations = {
     'download': 'Download PEM',
     'noRoas': 'Keine ROAs gefunden.',
     'noResources': 'Sie haben noch keine Ressourcen erhalten',
-    'noChildren': 'Keine Kinder gefunden.',
+    'noChildren': 'Keine untergeordneten CAs gefunden.',
```
Comment (partim) [inline src/locales/de.ts:69]: For consistency, I suppose “CAs“ should “Zertifizierungsstellen” here? (There’s a few more of those further down …)

File: src/locales/de.ts [hunk 2]
```diff
@@ -219,16 +219,16 @@ export const translations: Translations = {
...
-    'ca_parent_response_wrong_xml': 'Datenbank (Repository) Antwort während des Hinzufügens der Eltern-Zertifizierungsstelle erhalten',
-    'ca_child_duplicate': 'Es existiert bereits eine Kinder-Zertifizierungsstelle mit dem Namen \'{child}\'',
-    'ca_child_unknown': 'Es existiert keine Kinder-Zertifizierungsstelle mit dem Namen \'{child}\'',
-    'ca_child_resources_required': 'Sie müssen Ressourcen für die Kinder-Zertifizierungsstelle angeben \'{child}\'',
+    'ca_repo_response_wrong_xml': 'Antwort der übergeordneten Zertifizierungsstelle anstatt der Datenbank (Repository) erhalten',
+    'ca_parent_duplicate': 'Es existiert bereits eine übergeordnete Zertifizierungsstelle mit dem Namen \'{parent}\'',
+    'ca_parent_xml_duplicate': 'Diese Antwort XML ist bereits in Gebrauch von einer übergeordneten Zertifizierungsstelle mit dem Namen \'{parent}\'',
+    'ca_parent_unknown': 'Es existiert keine übergeordnete Zertifizierungsstelle mit dem Namen \'{parent}\'',
+    'ca_parent_issue': 'Error von übergeordneter Zertifizierungsstelle \'{parent}\' empfangen: {cause}',
+    'ca_parent_response_invalid_xml': 'Ungültiges xml von übergeordneter Zertifizierungsstelle empfangen',
```
Comment (partim) [inline src/locales/de.ts:227]: “xml” here in capitals, i.e., “XML”?

File: src/locales/de.ts (hunk 1 above)
Comment (partim) [reply src/locales/de.ts]: Aye, makes sense!

**Comment by timbru** [comment]
This will be released next week - we need to do a 0.14.3 release anyway to address a corner case issue.

**Pull request merged**